Alignment/HIPAlignmentAlgorithm/python/common_cff_py.txt

```diff
@@ -1,8 +1,25 @@
 from CondCore.DBCommon.CondDBSetup_cfi import *
 
 # loading magnetic field and geometry
+
+#process.load('Configuration.Geometry.GeometryExtended_cff')
+#process.TrackerTopologyEP = cms.ESProducer("TrackerTopologyEP")
```

Comment:
@meng-xiao there are many commented lines with obsolete configuration. Is it possible to clean them up?